blog/2019-12-18-debezium-1-0-0-final-released.adoc

```diff
@@ -0,0 +1,140 @@
+Debezium is deployed to production at lots of organizations from all kinds of industries,
+partly with huge installations, using hundreds of connectors to stream data changes out of thousands of databases.
+
+The 1.0 release marks an important milestone for the project:
+based on all the production feedback we got from the users of the 0.x versions, we figured it's about time to express the maturity of the four stable connectors in the version number, too.
+
+== Why Debezium?
+
+One of the things making it so enjoyable to work on Debezium as a tool for change data capture is the variety of potential use cases.
+When presenting the project at conferences,
+it's just great to see how people quickly get excited when they realize all the possibilities enabled by Debezium and CDC.
+
+In a nutshell, Debezium is one big enabler for letting you react to changes in your data with a low latency.
+Or, as one conference attendee recently put it, it's "like the observer pattern, but for your database".
+
+Here's a few things we've seen Debezium being used for as a ingestion component in data streaming pipelines:
+
+* Replicating data from production databases to other databases and data warehouses
+* Feeding data to search services like Elasticsearch or Apache Solr
+* Updating or invalidating caches
+
+When using Debezium with Apache Kafka and its rich ecosystem of sink connectors
+setting up such integrations can be done without any coding,
+just by means of configuration.
+
+But there's many other use cases of CDC which go beyond just moving data from A to B.
+When adding stream processing into the picture, e.g. via Kafka Streams or Apache Flink,
+CDC enables you to run https://github.com/debezium/debezium-examples/tree/master/kstreams-live-update[time-windowed streaming queries], continuously updated as your data changes
+("what's the aggregated order revenue per category within the last hour").
+You can use CDC to link:/blog/2019/10/01/audit-logs-with-change-data-capture-and-stream-processing/[build audit logs] of your data,
+telling who did change which data items at which time.
+Or update denormalized views of your data, for the sake of efficient data retrieval, adhering to the CQRS pattern.
+
+Finally, CDC also can play a vital role in microservices architectures;
+exchanging data between services and keeping local views of data owned by other services achieves a higher independence, without having to rely on synchronous API calls.
+One particularly interesting approach in this context is the link:/blog/2019/02/19/reliable-microservices-data-exchange-with-the-outbox-pattern/[outbox pattern],
+which is link:/documentation/reference/1.0/configuration/outbox-event-router.html[supported well] by Debezium.
+In case you don't start on the green field (who ever does?),
+CDC can be used to implement the strangler pattern for moving from a monolithic design to microservices.
```

Comment:
link to something describing this pattern?

Reply:
👍  Done.